templates/taxi/manufacturer_list.html

```diff
@@ -21,30 +24,31 @@ <h1>
       {% for manufacturer in manufacturer_list %}
         <tr>
           <td>
-              {{ manufacturer.id }}
+            {{ manufacturer.id }}
           </td>
           <td>
-              {{ manufacturer.name }}
+            {{ manufacturer.country }}
```

Comment:
The 'country' field is incorrectly placed in the 'Name' column. It should display 'manufacturer.name' instead.